tox.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = py26,py27,lint,docs
```

Comment:
I dropped py26 in there as we've so far maintained backwards compatibility, though we aren't running tests on travis for py26. I didn't realize 1.7 dropped py26 support though, so this can likely be dropped